### DISCUSSION TITLE
Tag created by API does not trigger `create`

### DISCUSSION THREAD

**Discussion by reitzig** [community/community#25034] [Actions]
<p>The docs say:</p>
<blockquote>
<p>Runs your workflow anytime someone creates a branch or tag, which triggers the create event. For information about the REST API, see “<a href="https://developer.github.com/v3/git/refs/#create-a-reference" rel="nofollow noopener">Create a reference</a>” in the GitHub Developer documentation.</p>
</blockquote>
<p>And yet, when I <a href="https://github.com/reitzig/texlive-docker/blob/master/.github/scripts/make-release-tag.sh" rel="nofollow noopener">do just that</a>, a workflow with the create trigger is not run.</p>
<pre><code>on:
create:
</code></pre>
<p>The workflow <em>is</em> run when I push the tag via my local Git CLI.</p>
<p>Did I misunderstand something?</p>
<p>Reproducing repo: <a href="https://github.com/reitzig/texlive-docker/tree/2437136b5e347ce6c868906bcaf44eb25caa7fb6" rel="nofollow noopener">https://github.com/reitzig/texlive-docker</a></p>

**Comment by dawidd6** [comment]
<p>Workflows triggered by github-actions are not run. And I see your tags are created by this user.</p>

**Comment by reitzig** [comment]
<blockquote>
<p>Workflows triggered by github-actions are not run.</p>
</blockquote>
<p>Is that documented anywhere?</p>
<p>And what’s the alternative to create DRY (= don’t repeat test jobs in multiple workflows) and clean (= don’t put everything in one file) workflows?</p>
<blockquote>
<p>And I see your tags are created by this user.</p>
</blockquote>
<p>Not sure where you see that?</p>
<p>If I understand correctly, a workaround would be to send the API requests with credentials for my Github user instead of using GITHUB_TOKEN?</p>

**Comment by reitzig** [comment, marked as answer]
<p>I can confirm that using an access token for the “real” user is a workaround. Thanks for the hint!</p>

**Comment by tinesoft** [comment]
<p>Hi,</p>
<p>I have excatly the same issue on my <a href="https://github.com/tinesoft/ngx-uptodate/" rel="nofollow noopener">project</a> : a <a href="https://github.com/tinesoft/ngx-uptodate/blob/develop/.github/workflows/release.yml" rel="nofollow noopener">first worklow</a> creates a tag, that should trigger the execution of the <a href="https://github.com/tinesoft/ngx-uptodate/blob/develop/.github/workflows/publish.yml" rel="nofollow noopener">second worklow</a>. But that never happens… Everything works fine when i mannually create and push the tag myself…</p>
<p>I <a href="https://github.com/tinesoft/ngx-uptodate/blob/0bb5320d45005fcc4416e6a3cbac44bceca5df5e/.github/workflows/release.yml#L17" rel="nofollow noopener">DID set a Personal Access Token</a> ( instead of Actions’s default GITHUB_TOKEN)  as value for the GITHUB_TOKEN used by workflow 1, but still, it not working…</p>
<p>Any help?</p>

**Comment by agronholm** [comment]
<blockquote>
<p><a class="mention" href="/dawidd6">@dawidd6</a> wrote:<br>
Workflows triggered by github-actions are not run. And I see your tags are created by this user.</p>
</blockquote>
<p>I have no idea what this means, but I am having the same problem: my main workflow creates a tag and that is supposed to trigger another workflow but it doesn’t happen unless I push the tag myself. I’m using the “create” trigger. Is there anything about this in the docs that I might have missed?</p>
